truffle.js

```diff
@@ -16,14 +16,15 @@ module.exports = {
       host: 'localhost',
       port: 8545,
       network_id: '*',
-      gasPrice: 100000000000
+      gas: 0x5B8D80, // 6000000 gas
+      gasPrice: 21000000000 // 21 Gwei
```

Comment:
if we change the gasPrice here, we also have to change it in `helpers.js`

Reply:
:+1: